distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -12,6 +12,7 @@ The ${command-console} has a command-line option for ingesting data.
 [NOTE]
 ====
 Ingesting with the console ingest command creates a metacard in the catalog, but does not copy the resource to the content store.
+The Ingest Command requires read access to the directory being ingested from, see the <<_url_resource_reader, URL Resource Reader>> for configuring read permissions on the directory.
```

Comment:
Can you update the link to `<<{architecture-prefix}url_resource_reader, URL Resource Reader>>`? There are some pending changes to the docs & docs test that will require it

Reply:
okay thanks for the feedback. Please feel free to review again @ricklarsen